src/components/FileList.tsx

```diff
@@ -110,93 +107,119 @@ export class FileList extends React.Component<IFileListProps, IFileListState> {
       disableFiles: false
     };
 
-    /** Add right-click menu options for files in repo
-     *
-     */
-
-    if (!commands.hasCommand(CommandIDs.gitFileOpen)) {
```

Comment:
Why are the `if` statements no longer required?

Reply:
It was left-over from the existing implementation. Commands have unique IDs so this isn't required.  Validated by checking examples in the `jupyterlab` repo

https://github.com/search?utf8=%E2%9C%93&q=%22commands.addCommand%22+user%3Ajupyterlab&type=Code&ref=advsearch&l=&l=